thrust/testing/pair.cu

```diff
@@ -322,4 +322,16 @@ void TestPairStructuredBindings(void)
   ASSERT_EQUAL(b, b2);
 }
 DECLARE_UNITTEST(TestPairStructuredBindings);
-#endif
+
+void TestPairCTAD(void)
```

Comment:
Question: any reason to keep the `void` other than (very) old habits? We are not aiming to compile with C?
```suggestion
void TestPairCTAD()
```
Second occurance later.